chain/network/src/peer_manager/peer_store/mod.rs

```diff
@@ -592,12 +594,12 @@ impl PeerStore {
 }
 
 /// Public method used to iterate through all peers stored in the database.
-pub fn iter_peers_from_store<F>(store: near_store::NodeStorage, f: F)
+pub fn iter_peers_from_store<F>(db: Arc<dyn Database>, f: F)
```

Comment:
Note for reviewers:
This is used by the state-view command where I want to select what store (hot/ cold) to use and only pass in the store of chosen temperature.